src/components/icon/Icon.mdx

```diff
@@ -1,122 +0,0 @@
-import { Meta, Story, Canvas, Controls } from '@storybook/blocks';
-import * as IconStories from './Icon.stories.js';
-import { stringToReactHtml } from '../../common/helpers/storybook';
-
-import logoutIcon from '../../assets/svg/logout.svg';
-import kyndrylIcon from '../../assets/svg/kyndryl-logo.svg';
-import bridgeSmallIcon from '../../assets/svg/bridge-logo-small.svg';
-import bridgeLargeIcon from '../../assets/svg/bridge-logo-large.svg';
-import switcherIcon from '../../assets/svg/switcher.svg';
-
-<Meta of={IconStories} />
-
-# Icons
-
-<br />
-
-## Carbon Icons
-
-Most of our icon usage will come from IBM Carbon.
-
-### Framework-specific Icons
-
-It is recommended to use framework-specific versions of the Carbon components.
-
-- [React](https://github.com/carbon-design-system/carbon/tree/main/packages/icons-react)
-- [Svelte](https://github.com/carbon-design-system/carbon-icons-svelte)
-- [Vue](https://github.com/carbon-design-system/carbon/tree/main/packages/icons-vue)
-- [Angular](https://github.com/carbon-design-system/carbon-icons-angular)
-
-### Vanilla Icons
-
-Use the `<kd-icon>` component to make using [vanilla Carbon Icons](https://github.com/carbon-design-system/carbon/tree/main/packages/icons) (bundled with this library) easy. This is more intended for vanilla/Lit usage internally, but should work anywhere.
-
-```js
-import '@kyndryl-design-system/shidoka-foundation/components/icon';
-import userAvatarIcon from '@carbon/icons/es/user--avatar/24';
-```
-
-```html
-<kd-icon .icon="${userAvatarIcon}"></kd-icon>
-```
-
-<Canvas>
-  <Story of={IconStories.Icon} />
-</Canvas>
-
-<Controls of={IconStories.Icon} />
-
-### Gallery
-
-[Carbon Icon Library](https://carbondesignsystem.com/guidelines/icons/library/)
-
-<br />
-
-## Kyndryl Icons
-
-Kyndryl icons are custom SVG files that you can import directly from this library.
-
-### Example Usage
-
-It is recommended to use an SVG loader to inject the SVG as inline HTML into your app.
-
-- Example Loaders:
-  - [Rollup](https://www.npmjs.com/package/rollup-plugin-svg)
-  - [Webpack](https://www.npmjs.com/package/svg-inline-loader)
-
-SVG files can then be imported like this:
-
-```js
-import iconName from '@kyndryl-design-system/shidoka-foundation/assets/svg/<filename>';
-```
-
-The method for placing them in your app at this point will vary based on the framework/loader.
-
-<br />
-
-### Gallery
```

Comment:
One small problem here, we still need to document the usage of the logo SVG assets somewhere. Logout and switcher can be removed.